task8/YM_Market_App/src/main/java/CollectionFolder/AddCollection.java

```diff
@@ -0,0 +1,97 @@
+package CollectionFolder;
+import CollectionFolder.AddClass;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+
+public class AddCollection {
+    private static List<AddClass> collection;
```

Comment:
Use static initialization for static fields. Do not initialize static fields in constructors or other instance methods because this is not thread-safe